adjacent_material.md

```diff
@@ -186,15 +184,18 @@ Doodles)](https://www.youtube.com/watch?v=hfMk-kjRv4c)
 
 ## Stable Diffusion
 
+#### How Stable Diffusion works
+
 In addition to Jeremy Howard's explanation we found that it was worth
```

Comment:
The name 'Jeremy Howard' is still mentioned here. Do you want me to remove it?

Reply:
I missed that one :)